### PR TITLE
zh: add some introduction in `overview.md`

### PR DESCRIPTION
**Pull request by IANTHEREAL**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
add some introduction in overview.md

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/overview.md [hunk 1]
```diff
@@ -5,7 +5,18 @@ aliases: ['/docs-cn/tidb-data-migration/dev/overview/','/docs-cn/tools/dm/overvi
 
 # Data Migration 简介
 
-[TiDB Data Migration](https://github.com/pingcap/dm) (DM) 是一体化的数据迁移任务管理工具，支持从与 MySQL 协议兼容的数据库（MySQL、MariaDB、Aurora MySQL）到 TiDB 的数据迁移。DM 工具旨在降低数据迁移的运维成本。
+[TiDB Data Migration](https://github.com/pingcap/dm) (DM) 是一体化的数据迁移任务管理工具，支持从与 MySQL 协议兼容的数据库（MySQL、MariaDB、Aurora MySQL）到 TiDB 的数据迁移。DM 工具旨在降低数据迁移的运维成本。 当你使用 DM 进行数据迁移的时候，需要执行下面的操作
```
Comment (CharLotteiu) [inline zh/overview.md:8]:
```suggestion
[TiDB Data Migration](https://github.com/pingcap/dm) (DM) 是一体化的数据迁移任务管理工具，支持从与 MySQL 协议兼容的数据库（MySQL、MariaDB、Aurora MySQL）到 TiDB 的数据迁移。DM 工具旨在降低数据迁移的运维成本。使用 DM 进行数据迁移的时候，需要执行以下操作：
```

File: zh/overview.md [hunk 2]
```diff
@@ -5,7 +5,18 @@ aliases: ['/docs-cn/tidb-data-migration/dev/overview/','/docs-cn/tools/dm/overvi
...
+
+- 部署 DM 集群
+- 创建上游数据源（source）对象，保存数据源访问信息
+- 创建（多个）数据迁移任务从数据源迁移数据到 TiDB
+
+数据迁移任务包含全量数据迁移、增量数据迁移两个阶段：
```
Comment (CharLotteiu) [inline zh/overview.md:14]:
```suggestion
数据迁移任务包含全量数据迁移、增量数据复制两个阶段：
```
参考 DM 文档[术语表](https://docs.pingcap.com/zh/tidb-data-migration/stable/glossary#%E5%A4%8D%E5%88%B6%E5%A2%9E%E9%87%8F%E5%A4%8D%E5%88%B6)

File: zh/overview.md [hunk 3]
```diff
@@ -5,7 +5,18 @@ aliases: ['/docs-cn/tidb-data-migration/dev/overview/','/docs-cn/tools/dm/overvi
...
+
+- 全量数据迁移：从数据源迁移对应表的表结构到 TiDB，然后读取存量数据写入到 TiDB 集群；
+- 增量数据迁移：全量数据迁移完成后，从数据源读取对应的表变更然后写入到 TiDB 集群。
```
Comment (CharLotteiu) [inline zh/overview.md:17]:
```suggestion
- 增量数据复制：全量数据迁移完成后，从数据源读取对应的表变更然后写入到 TiDB 集群。
```

File: zh/overview.md [hunk 4]
```diff
@@ -5,7 +5,18 @@ aliases: ['/docs-cn/tidb-data-migration/dev/overview/','/docs-cn/tools/dm/overvi
...
+
+下面介绍 DM 所具备的功能。
```
Comment (CharLotteiu) [inline zh/overview.md:19]:
```suggestion
下文介绍 DM 所具备的功能。
```

File: zh/overview.md [hunk 5]
```diff
@@ -58,11 +69,11 @@ DM 支持对源数据的分库分表进行合并迁移，但有一些使用限
...
     - DM 遇到不兼容的 DDL 语句时会报错。要解决此报错，需要使用 dmctl 手动处理，要么跳过该 DDL 语句，要么用指定的 DDL 语句来替换它。详见[如何处理不兼容的 DDL 语句](faq.md#如何处理不兼容的-ddl-语句)。
 
-+ 分库分表
++ 分库分表数据冲突合并
 
     - 如果业务分库分表之间存在数据冲突，可以参考[自增主键冲突处理](shard-merge-best-practices.md#自增主键冲突处理)来解决；否则不推荐使用 DM 进行迁移，如果进行迁移则有冲突的数据会相互覆盖造成数据丢失。
     - 分库分表 DDL 同步限制，参见[悲观模式下分库分表合并迁移使用限制](feature-shard-merge-pessimistic.md#使用限制)以及[乐观模式下分库分表合并迁移使用限制](feature-shard-merge-optimistic.md#使用限制)。
 
-+ 同步的 MySQL 实例变更
++ 同步的 MySQL 实例
```
Comment (CharLotteiu) [inline zh/overview.md:77]: 建议`使用限制`下面的这几点描述结构上统一一下，比如都换成名词+动词结构，例如：
- 数据库版本要求
- DDL 语法兼容
- 分库分表数据冲突合并
- MySQL 实例同步切换

**Review by lance6716 — APPROVED** [review]

**Comment by IANTHEREAL** [comment]
@CharLotteiu PTAL

**Review by CharLotteiu — APPROVED** [review]

**Pull request merged**